sf-stream/src/metrics.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{register_int_counter, IntCounter};
+use once_cell::sync::Lazy;
+
+pub static TRANSACTIONS_SENT: Lazy<IntCounter> = Lazy::new(|| {
```

Comment:
We probably want to log the transaction id explicitly right?

Reply:
that's super high cardinality; generally high cardinality + Prometheus = sad Prometheus